### DISCUSSION TITLE
Create path similar to /{namespace}/{pod-name}/{container-name}/*.log to store logs using file output plugin.

### DISCUSSION THREAD

**Discussion by maitry-nutanix** [fluent/fluent-bit#7386] [Q&A]
I want to store logs that are being read from tail input plugin and store them in the destination by creating a directory structure for them. Currently all the logs are stored in the Destination folder with name as kube.var.log.pods.namespace.pod_name.id.container_name.0.log. Is it possible to create a directory structure for storing logs inside destination directory as path_to_destination//{namespace}/{pod-name}/{container-name}/*.log 

Until now I've tried to use lua filter but didn't get much success with it because there's nothing like record["path"] in which I can specify the path where to store my logs. Also, dynamic folder creation in the path key of output plugin is not supported. 
Need help in how to do that.

**Comment by patrick-stephens** [comment]
From memory, I think I did this previously by rewriting the tag to match what I wanted.

**Reply by maitry-nutanix** [reply]
can you elaborate ?

**Reply by patrick-stephens** [reply]
I used rewrite_tag to just add slashes into the tag for the directories I wanted and left the file output plugin to use it that way. However it was a while ago I did this and I don't have an example to hand.

This was on Linux too.
